forge-game/src/main/java/forge/game/card/Card.java

```diff
@@ -5182,6 +5182,15 @@ public final boolean hasStartOfUnHiddenKeyword(String keyword, CardState state)
         return visitor.getResult();
     }
 
+    public final boolean hasTriggerContaining(final String value) {
+        for (Trigger t: this.getTriggers()) {
+            if (t.toString().contains(value)) {
```

Comment:
You would have to implement a better trigger parsing than that

This will only work when user plays in english

Reply:
I've switched to checking for trigger types.